src/sssom/util.py

```diff
@@ -1101,6 +1103,11 @@ def is_curie(string: str) -> bool:
 
 def is_iri(string: str) -> bool:
```

Comment:
Missing `@deprecated` annotator?

Reply:
I don't think there are any remaining instances of using `@deprecated` in this package. Is there a benefit to using that instead of just giving a clear warning?

Reply:
Yes, documentation generation! Also, technically, `@deprecated` should automatically produce the warning you added manually, leading to less code clutter

Reply:
Done in 1654b4a